.github/workflows/test.yml

```diff
@@ -293,3 +329,12 @@ jobs:
             ${{ steps.reports.outputs.path }}/${{ matrix.name }}.xml
             ${{ steps.reports.outputs.path }}/${{ matrix.name }}.json
         continue-on-error: true
+
+      - name: Create issue on failure
+        if: failure() && github.event_name == 'schedule'
+        uses: ipdxco/create-or-update-issue@0265e9148d6c26f6cdb0abb9c47df048a585c2fe # v1.0.0
```

Comment:
Is it possible to do a @1.0.0 now that there is a 1.0 release?  (I don't know how this stuff works - just curious)

Reply:
You could, but the recommended practice here is to pin the hash instead of the (mutable) version tag.

Github security guidelines on the subject: [pin action section](https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions#using-third-party-actions)

Reply:
Got it - thanks.  I thought I saw in some of these configs we did use @version (or maybe that is just with the github provided actions which I guess we can trust :) )